src/test/java/Test1.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Test1 {
```

Comment:
why do you need so much test classes? Why don't you place your tests in one class? what is the logic here?

Reply:
These tests arent connected anyway, so that`s why i desided to split between classes